push_notifications/models.py

```diff
@@ -56,7 +56,7 @@ def get_queryset(self):
 
 class GCMDeviceQuerySet(models.query.QuerySet):
 	def send_message(self, message, **kwargs):
-		if self:
+		if self.exists():
```

Comment:
checking `queryset.exists()` is more efficient than checking queryset truthiness [read more](https://django.doctor/advice/C9003)